lgc/interface/lgc/GpurtDialect.td

```diff
@@ -107,3 +109,183 @@ def GpurtGetTriangleCompressionModeOp : GpurtOp<"get.triangle.compression.mode",
   let results = (outs I32:$result);
   let summary = "return the traceRay triangle compression mode";
 }
+
+def GpurtGetFlattenedGroupThreadIdOp : GpurtOp<"get.flattened.group.thread.id", [Memory<[]>, WillReturn]> {
+  let arguments = (ins);
+  let results = (outs I32:$result);
+  let summary = "return the flattened group thread ID";
+}
+
+def GpurtSetHitAttributesOp : GpurtOp<"set.hit.attributes", [Memory<[(write)]>, WillReturn]> {
```

Comment:
This and the other `set.foo` operations can be `Memory<[(write InaccessibleMem)]>`, which is slightly more precise.